examples/nextjs-with-typescript/src/ProTip.tsx

```diff
@@ -25,7 +25,7 @@ const useStyles = makeStyles((theme: Theme) =>
 );
 
 export default function ProTip() {
-  const classes = useStyles();
+  const classes = useStyles({});
```

Comment:
This is incorrect, `useStyles` doesn't want props here, the tsconfig shouldn't set strict to false 

cc @eps1lon

Reply:
Ah right, this was an open issue I still had to look into, was getting:
```
28:19 Expected 1 arguments, but got 0.
    26 | 
    27 | export default function ProTip() {
  > 28 |   const classes = useStyles();
       |                   ^
    29 |   return (
    30 |     <Typography className={classes.root} color="textSecondary">
    31 |       <LightBulbIcon className={classes.lightBulb} />
```